python/tvm/target/target.py

```diff
@@ -724,6 +724,12 @@ def create_llvm_options(cpu_ver, config):  # pylint: disable=unused-argument
 
         llvm_options = config["llvm_options"]
 
+        # To enable auto-vectorization for v68 target added the below llvm-option by default
+        if arch_version == 68:
+            if not llvm_options:
+                llvm_options = ""
+            llvm_options += " -force-hvx-float"
+
```

Comment:
Does this flag impose any new requirements regarding which LLVM versions TVM needs?

Reply:
Also, are there any potential downsides or other side-effects of using this flag?  Asking because with compiler flags, the word "force" sometimes implies that the resulting behavior isn't always a good idea.

Reply:
It will work with any LLVM version that supports v69.  It's fine to use it.